File: backend/src/Outlook.Models/Core/Models/Issue.cs
using Outlook.Models.Core.Interfaces;
using System.Collections.Generic;
using System.ComponentModel;

namespace Outlook.Models.Core.Models
{
    public class Issue : IIssue<Article>
    {
        public int Id { get; set; }

        public int Number { get; set; }

        public Volume Volume { get; set; }

        [DisplayName("Arabic Theme")]
        public string ArabicTheme { get; set; }
        
        [DisplayName("English Theme")]
        public string EnglishTheme { get; set; }

        public List<Article> Articles { get; set; }

        public Issue SetIssueNumber(int number)
        {
            Number = number;
            return this;
        }

        public Issue SetArabicTheme(string theme)
        {
            ArabicTheme = theme;
            return this;
        }

        public Issue SetEnglishTheme(string theme)
        {
            EnglishTheme = theme;
            return this;
        }
    }
}

File: mobile/lib/models/issue.dart
import 'package:flutter/cupertino.dart';
import 'package:mobile/models/volume.dart';

class Issue {
  int id;
  int number;
  String arabicTheme;
  String englishTheme;
  Volume volume;

  Issue({
    @required this.id,
    @required this.number,
    @required this.arabicTheme,
    @required this.englishTheme,
    @required this.volume
  });

  Issue.fromJson(Map<String, dynamic> json) :
    id = json['id'],
    number = json['number'],
    arabicTheme = json['arabicTheme'],
    englishTheme = json['englishTheme'],
    volume = (json['volume'] != null)? Volume.fromJson(json['volume']) : null;
}